conf/messages

```diff
@@ -1353,3 +1353,17 @@ itemCheckAnswers.wineDetailsCardTitle = Wine details
 itemCheckAnswers.packagingCardTitle = Packaging
 itemCheckAnswers.packageTypeKey = Packaging type {0}
 itemCheckAnswers.packageTypeValue = {0}x {1}
+
+itemsAddToList.title.singular = You have given information for 1 item
+itemsAddToList.title.plural = You have given information for {0} items
+itemsAddToList.heading.singular = You have given information for 1 item
+itemsAddToList.heading.plural = You have given information for {0} items
+itemsAddToList.itemCardTitle = Item {0}
+itemsAddToList.packagesCyaLabel = Packaging
```

Comment:
The copydeck has a 'Quantity' entry. I haven't ran the code yet but should there be a 'Quantity' section or something similar?